### DISCUSSION TITLE
Populating additional relations

### DISCUSSION THREAD

**Discussion by andrewscfl** [strapi-community/plugin-io#25] [Ideas]
It would be nice if there was a way to specify population of additional relations beyond the main event target collection and its immediate relations. We're using the plugin to sync live information on a table and one of the fields is a part of a collection attached to a linked collection. so like TableA -> TableB -> TableC could all be populated on the update response.

As of right now we're using the socket event to trigger a REST call so that all fields are populated but could be a helpful feature to add in the future.

**Comment by ComfortablyCoding** [comment]
Any sort of implementation we do will be as performant as doing the secondary REST call that you are currently doing. Auto populating relations will require a call to the resource which is equivalent to a REST call. Leaving it to the REST call also provides the user with more control over the data processing.

If you have any suggestions onalternatives they are welcome.
